openedx/core/djangoapps/programs/utils.py

```diff
@@ -326,3 +333,32 @@ def _attach_course_run_upgrade_url(self, run_mode):
                 run_mode['upgrade_url'] = None
         else:
             run_mode['upgrade_url'] = None
+
+    def _attach_instructors(self):
+        """
+        Extend the program data with instructor data. The instructor data added here is persisted
+        on each course in modulestore and can be edited in Studio. Once the course metadata publisher tool
+        supports the authoring of course instructor data, we will be able to migrate course
+        instructor data into the catalog, retrieve it via the catalog API, and remove this code.
+        """
+        cache_key = 'program.instructors.{uuid}'.format(
+            uuid=self.data.get('uuid')
```

Comment:
Use of `get()` here is unnecessarily defensive, just do `self.data['uuid']`. Well-formed programs will always include a UUID.